features/steps/steps.py

```diff
@@ -428,9 +423,10 @@ def then_stream_does_not_match_regexp(context, stream):
     assert_that(content, not_(matches_regexp(context.text)))
 
 
-@then("stderr matches regexp")
-def then_stderr_matches_regexp(context):
-    assert_that(context.process.stderr.strip(), matches_regexp(context.text))
+@then("{stream} matches regexp")
+def then_stream_matches_regexp(context, stream):
```

Comment:
+1 had wanted to fix that as well in an earler commit/review comment. Thanks for taking it to the next step.